src/bundler.ts

```diff
@@ -24,6 +25,35 @@ interface BundleOptions {
   onBeforeDownload?: LifecycleHook
 }
 
+interface WriteResultsOptions {
+  bundles: Bundle[]
+  declarations: Declaration[]
+  distDirectory: string
+  functions: EdgeFunction[]
+  distImportMapPath?: string
+  importMap?: ImportMap
+}
+
+const writeResults = async ({
+  bundles,
+  declarations,
+  distDirectory,
+  functions,
+  distImportMapPath,
+  importMap,
+}: WriteResultsOptions) => {
+  await writeManifest({
+    bundles,
+    declarations,
+    distDirectory,
+    functions,
+  })
+
+  if (distImportMapPath && importMap) {
+    await importMap.writeToFile(distImportMapPath)
+  }
+}
+
```

Comment:
How do you feel about it? I would actually prefer to ignore the rule in this case, because it feels a bit arbitrary to break this bit out into a separate function just to satisfy the linter, and I think it actually makes the code a bit more difficult to follow.

Reply:
I definitely agree, wasn't sure how we felt about ignoring the linter though, but there didn't really seem to be a particularly clean split anywhere. I'll update to ignore it for this function